cluster-cm/src/test/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerLdapServiceTest.java

```diff
@@ -111,6 +119,18 @@ public void testSetupLdapWithNoRoleAdmin() throws ApiException {
 
     @Test
     public void testSetupLdapWithoutLdap() throws ApiException {
+        // GIVEN
```

Comment:
This test is redundant with testSetupLdapWithTrial. The only difference is that the latter one doesn't check on the trial license. They should be recombined, or else they should test different conditions.

Reply:
Will just go back to the original test + the beginTrial check.